app/models/customer.py

```diff
@@ -0,0 +1,29 @@
+from app import db
+from flask import current_app
+from sqlalchemy import DateTime
+from sqlalchemy.orm import relationship, backref
```

Comment:
we already have access to these from the `db` import. That's why we have `db` before each data type, like `db.Integer`. WIthout important the instance of our database from `__init__.py`, *then* we would need to import these modules here

```suggestion

```